### PR TITLE
 Add metrics/logging and REST endpoint for LocalScheduler

### PR DESCRIPTION
**Pull request by tbak**
### Description of the Change

Add metrics/logging and REST endpoint for LocalScheduler.

### CODE REVIEW

File: titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/model/TransactionId.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package com.netflix.titus.common.framework.scheduler.model;
+
+import java.util.Objects;
+
+public class TransactionId {
+
+    private static final TransactionId INITIAL = new TransactionId(1, 0, 1);
+
+    private final int major;
```
Comment (corindwyer) [inline titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/model/TransactionId.java]: I think the current object model is confusing. Talked offline for other approaches.

File: titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/internal/LocalSchedulerTransactionLogger.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        String summary;
+        switch (schedulingState) {
+            case Waiting:
+                summary = "Waiting...";
+                break;
+            case Running:
+                summary = "Running...";
+                break;
+            case Cancelling:
+                summary = "Schedule canceled by a user";
```
Comment (corindwyer) [inline titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/internal/LocalSchedulerTransactionLogger.java]: canceled -> cancelled

File: titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/model/TransactionId.java (hunk 1 above)
Comment (tbak) [reply titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/model/TransactionId.java]: Changed

File: titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/internal/LocalSchedulerTransactionLogger.java (hunk 2 above)
Comment (tbak) [reply titus-common/src/main/java/com/netflix/titus/common/framework/scheduler/internal/LocalSchedulerTransactionLogger.java]: Fixed

**Pull request merged**